cpu/sam0_common/include/sam_usb.h

```diff
@@ -0,0 +1,74 @@
+/*
+ * Copyright (C) 2019 Koen Zandberg
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @defgroup    cpu_sam0_common_usb sam0 USB peripheral
+ * @ingroup     cpu_sam0_common
+ * @brief       USB interface functions for the sam0 class devices
+ *
+ * @{
+ *
+ * @file
+ * @brief       USB interface functions for the sam0 class devices
+ *
+ * @author      Koen Zandberg <koen@bergzand.net>
+ */
+
+#ifndef SAM_USB_H
+#define SAM_USB_H
+
+#include <stdint.h>
+#include <stdlib.h>
+#include "periph_cpu.h"
+#include "periph/usbdev.h"
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/**
+ * USB endpoint buffer space
+ *
+ * @todo global configurable?
```

Comment:
still todo ?

Reply:
Nope, actually is globally configurable now (for a while)